server/core/terraform/async_client.go

```diff
@@ -134,3 +142,12 @@ func (c *AsyncClient) RunCommandAsyncWithInput(ctx context.Context, prjCtx comma
 
 	return outCh
 }
+
+func (c *AsyncClient) logLineShouldBeStripped(message string) bool {
+	for _, line := range c.logPrefixesToStrip {
+		if strings.HasPrefix(message, line) {
```

Comment:
Prefix seems oddly specific to this usecase.  I'd rather us use regex matching, this is probably all code we're going to keep so let's future proof it.